src/pybind/mgr/mgr_module.py

```diff
@@ -1239,7 +1251,7 @@ def send_command(
         """
         self._ceph_send_command(result, svc_type, svc_id, command, tag, inbuf)
 
-    def set_health_checks(self, checks):
+    def set_health_checks(self, checks: Dict[str, Dict[str, Union[str, int, List[str]]]]) -> None:
```

Comment:
fyi this relates to https://tracker.ceph.com/issues/48510

Reply:
ack.